### PR TITLE
Fixes #23292 - make s390x ks repos bootable

### PR DESCRIPTION
**Pull request by sean797**
https://github.com/Katello/katello/commit/715a11bd8eec7e08c8c6800449349ecfa6dd5fbe
makes newly created s390x repos 'bootable' but never migrated
any pre existing repos.

### CODE REVIEW

File: db/migrate/20180416160223_make_s390x_ks_repos_bootable.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class MakeS390xKsReposBootable < ActiveRecord::Migration[5.1]
+  def change
+    Katello::Repository.import_distributions
```
Comment (jlsherrill) [inline db/migrate/20180416160223_make_s390x_ks_repos_bootable.rb]: we have a rule to not call out to backend services as part of a migration unless we absolutely have to.   An upgrade rake task would be more appropriate here

Comment (sean797) [reply db/migrate/20180416160223_make_s390x_ks_repos_bootable.rb]: I'll move it to `lib/katello/tasks/upgrades/3.7/` :+1: 

How else do you want to do it? If we want to check this in the most complete way, we need to reach out to pulp for the filenames (https://github.com/Katello/katello/commit/715a11bd8eec7e08c8c6800449349ecfa6dd5fbe#diff-f6aa9601a7c1afa89d06151a45d30777R58)

We could do something like this if you like?
```
Katello::Repository.where(:distribution_arch => 's390x').where('name ilike ?', '%kickstart%').update_all(:distribution_bootable => 'true')
```
Maybe instead of name we could also use `cp_label` or `label`, but either way someone could have created a custom repo with kickstart in the name or label. 

tldr;
I think we need to reach out to pulp to do this correctly.

Comment (jlsherrill) [reply db/migrate/20180416160223_make_s390x_ks_repos_bootable.rb]: oh yes, to do this we have to reach out to pulp, no other way to do it :)  which means we should do this outside of a migration.  

The only case where i  can think where we'd have to do reach out within a db migration is if we added some column that had required data (non-null column), but that data was only in a backend system.  I can't think of why that would ever occur, so we should be able to get away without it.

Comment (sean797) [reply db/migrate/20180416160223_make_s390x_ks_repos_bootable.rb]: right, I understand what you were saying now :-) Updated!
